docs/source/configuration.rst

```diff
@@ -80,6 +80,40 @@ PC15  N/A        N/A     N/A
 ====  =========  ======  ==============================
 
 
+.. index:: interruptions
+
+Exceptions
+==========
+
+The configuration for interruptions (IRQ) and SysTick exceptions is described below. For more information have a look at the `STM32F10xxx/20xxx/21xxx/L1xxxx Cortex®-M3 programming manual`_
```

Comment:
Missing `.` at the end of the paragraph. :wink:

Reply:
:+1: